Control/CreateUser/CreateUsers.go

```diff
@@ -1,26 +0,0 @@
-package CreateUser
```

Comment:
package name should be in lower case and should not be camelcase
(suggestion: create_user)